### ISSUE TITLE
About pretrain model！

### ISSUE THREAD

**Issue by zhangmo123** [happynear/AMSoftmax#10]
Hi thanks for you amsoftmax:
    Could you tell us if the ResNet20 is a pretrained model？ And if it is pretrained with the amsoftmax！

**Comment by happynear** [comment]
It is trained using AM-Softmax on CASIA-Webface from **scratch**. You may use it as a pre-trained model for other tasks.
